contracts/TokenConverterRoute.sol

```diff
@@ -0,0 +1,64 @@
+pragma solidity ^0.4.24;
+
+import "./interfaces/TokenConverter.sol";
+import "./interfaces/AvailableProvider.sol";
+import "./interfaces/Token.sol";
+import "./utils/Ownable.sol";
+import "./vendors/bancor/converter/BancorGasPriceLimit.sol";
+
+
+contract TokenConverterRoute is TokenConverter, Ownable {
+    
+    uint256 constant internal MAX_UINT = uint256(0) - 1;
+    TokenConverter[] private converters;
+    mapping (address => address) private availability;
+    
+    function addConverter(TokenConverter converter, AvailableProvider availabilityContract) onlyOwner public {
```

Comment:
This setter only allows to add new converters, there should exist the posibility of removing converters and adding/removing availability contracts.

Reply:
Done